src/component/addBook/AddBook.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { baseURL } from '../../helpers/api';
+import InputChange from './hooks/inputChange';
+import UserFileUpload from './hooks/userFileUpload';
+import Container from './reusable/container/Container';
+import FileUpload from './reusable/inputFields/FileUpload';
+import Input from './reusable/inputFields/input';
+import isUserSigned from '../../helpers/auth';
+
+const AddBook = () => {
```

Comment:
- [ ] When I want to add a picture to a book that I'm creating, if the image is high resolution it will overflow the frontend containers:

![image](https://user-images.githubusercontent.com/57421823/225775140-ff6e6074-fe5b-421d-9755-d94e0db62fdf.png)

This issue is more prominent in the mobile view.

Kindly, add some styling so the image has limits and it does not overflow. 👍

Reply:
done.